Installation/macOS.md

```diff
@@ -1,3 +1,16 @@
 # Installing on macOS
 
-We're working hard to bring Procursus to macOS. When it's ready for everyone to use, this page will be updated with information on how to get started.
+```zsh
+curl -LO https://cameronkatri.com/zstd
+Intel: curl -L https://apt.procurs.us/bootstrap_darwin-amd64.tar.zst -o bootstrap.tar.zst
+M1: curl -LO https://cdn.discordapp.com/attachments/763074782220517467/819588605999317022/bootstrap.tar.zst
+chmod +x zstd
+./zstd -d bootstrap.tar.zst
+sudo tar -xpkf bootstrap.tar -C /
+printf 'export PATH="/opt/procursus/bin:/opt/procursus/sbin:/opt/procursus/games:$PATH"\nexport CPATH="$CPATH:/opt/procursus/include"\nexport LIBRARY_PATH="$LIBRARY_PATH:/opt/procursus/lib"\n' | sudo tee -a /etc/zshenv /etc/profile
+export PATH="/opt/procursus/bin:/opt/procursus/sbin:/opt/procursus/games:$PATH"
+export CPATH="$CPATH:/opt/procursus/include"
+export LIBRARY_PATH="$LIBRARY_PATH:/opt/procursus/lib"
+sudo apt update
+sudo apt full-upgrade
+```
```

Comment:
These are not good instructions to give. Give actual instructions that explain what each step does.

Reply:
@CRKatri You wrote them? But sure, I'll close this PR and reopen once I "Give actual instructions that explain what each step does".

Reply:
@0x8ff A pinned message in a Discord server, for people that know what they are doing, is different than public documentation, that everyone is supposed to be able to understand. Just providing a script with no explanation whatsoever is just going to scare people away. No need to be rude to Cameron when you're the person that just copied and pasted a script instead of trying to write proper documentation.